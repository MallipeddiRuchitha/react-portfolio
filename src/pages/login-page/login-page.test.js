import React from 'react';
import { shallow ,mount} from 'enzyme';
import LoginPage from './login-page';
  
const auth={
    login:jest.fn()
}
describe("LoginPage component",() =>{
    test("Matches the snapshot",() => {
        const loginPage = shallow(< LoginPage/>);
        expect(loginPage).toMatchSnapshot();


    })
    test("LoginPage element is rendered",() => {
        const loginPage = shallow(< LoginPage/>);
        expect(loginPage.find('#loginPage').exists()).toEqual(true);
        
    });
   
   
   

});