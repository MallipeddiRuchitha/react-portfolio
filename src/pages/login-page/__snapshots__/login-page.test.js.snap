// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage component Matches the snapshot 1`] = `
<div
  id="loginPage"
>
  <LoginTemplate
    content={<Login />}
  />
</div>
`;
